src/main/kotlin/com/hibob/academy/employeeFeedback/resource/FeedbackResource.kt

```diff
@@ -38,7 +38,8 @@ class FeedbackResource(private val feedbackService: FeedbackService, private val
         val companyId = authenticatedUsersService.getLoggedInCompanyId(request)
         val role = authenticatedUsersService.getLoggedInRole(request)
 
-        if (!authenticatedUsersService.validateRole(role)) return Response.status(Response.Status.UNAUTHORIZED).build()
+        if (!authenticatedUsersService.validateHr(role) && !authenticatedUsersService.validateAdmin(role))
```

Comment:
so how will this look if I allow 2 more roles to access this api?

Your first instinct to do one validation function was good, then this code here is cleaner. But you need to make it more flexible

Reply:
fixed it